src/lib/rng/chacha_rng/chacha_rng.h

```diff
@@ -116,6 +164,14 @@ class BOTAN_PUBLIC_API(2, 3) ChaCha_RNG final : public Stateful_RNG {
 
       std::unique_ptr<MessageAuthenticationCode> m_hmac;
       std::unique_ptr<StreamCipher> m_chacha;
+      const bool m_fast_key_erasure;
+      const size_t m_chacha_keylen;
+
+      const static inline char* m_stream_cipher_algo = "ChaCha(20)";
+      const static inline char* m_hmac_algo = "HMAC(SHA-512)";
+
+      // use "classic" 8 byte nonce as extended key material
+      const static inline size_t m_chacha_iv_len = 8;
```

Comment:
Similarly as above: Please move to the top of the class and make it `constexpr size_t`